components/app/header/Banner.vue

```diff
@@ -18,13 +18,13 @@ function removeBanner() {
 </script>
 
 <template>
-  <div v-show="showBanner && bannerConfig?.show"
+  <div v-show="showBanner && bannerConfig && bannerConfig.show"
```

Comment:
Can also be `bannerConfig?.show` because javascript will pass `undefined` which is false ?

Reply:
Yeah, this is probably a memory muscle from my side, because not too long ago this was not possible in the template, 

Probably a computed property will be the best option

Reply:
Agree, computed is the best